mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationDir.java

```diff
@@ -38,7 +38,9 @@
 
 /**
  * This helper class determines in which directory to look for addition configuration files.
- * 
+ *
+ * The Configuration Directory can be set with the System property <code>MCR.ConfigDir</code>.
```

Comment:
The configuration directory can be set with the system property <code>MCR.ConfigDir</code>.